app/models/like.rb

```diff
@@ -1,5 +1,5 @@
 class Like < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User'
   belongs_to :post
 
```

Comment:
- [ ] Validation are missing for `CommentsCounter` and `LikesCounter`. Kindly ensure to add validation(s) for the `CommentsCounter` and `LikesCounter` such that they're an integer greater than or equal to zero.

Reply:
@rloterh thank you for your review. The reason they are missing is they were not required for the project and I only created the required ones. But I added them now according to your review. 

<img width="1440" alt="Screenshot 2022-10-28 at 15 57 37" src="https://user-images.githubusercontent.com/40325034/198596266-e507e1c3-1189-4378-9a33-e558ae905273.png">

Reply:
@bengigo 
This was addressed in the Post model. I apologize for missing this step. Cheers, and happy coding! 😃